bots/psuteam7bot/castle.js

```diff
@@ -13,30 +13,111 @@ castle.doAction = (self) => {
     {
         return castle.findUnitPlace(self, 'PILGRIM');
     }
-    else
+    else if(self.me.turn<100)
     {
         //Check if there are enough resources to produce this unit.
-       if(self.fuel >= SPECS.UNITS[SPECS.PROPHET].CONSTRUCTION_FUEL && self.karbonite >= SPECS.UNITS[SPECS.PROPHET].CONSTRUCTION_KARBONITE) {
+       if(self.fuel >= SPECS['PROPHET'].CONSTRUCTION_FUEL && self.karbonite >= SPECS['PROPHET'].CONSTRUCTION_KARBONITE){
            return castle.findUnitPlace(self, 'PROPHET');
        }
        return;
     }
-    return;
 
 }
 
 castle.findUnitPlace = (self, unitType) => {
     //Check if any of the adjacent tile is available. Place the unit if true.
-    for(let i=-1; i<= 1; i++){   
-        for(let j = -1; j<= 1; j++){
+    for(i= -1; i<= +1; i++){   
+        for(j = -1; j<= +1; j++){
```

Comment:
Should be `let i =` and `let j = `, and no `+` in `i <= +1`. Think the original version avoided these, so we can resolve to older version